### ISSUE TITLE
Changing the location of the covers and other news?

### ISSUE THREAD

**Issue by Miecz5** [eleybourn/Book-Catalogue#889]
Hello.
The new version surprised me because the connection with the covers disappeared and I didn't know what it was about? I restored the archive and in total I lost only (as many as) a dozen or so book covers entered a few days ago. I wouldn't lose any knowledge to move the photos to another place. I didn't expect that and I just deleted the cover base. I'll see if there's anything good in the new version 6.0.1 to use for me.
However, I noticed a different way of reading the archive in which there is no information about the number of books and the number of covers. Having a collection of several hundred books, such information was important to me because it showed the integrity of the size of the collection. It seems that in the new way of saving and reading the archive it will be impossible to put such information?
This may include information about the covers in the header of the database (e.g. displaying 652 books & 638 covers) and then I know that by the way I should fill in the missing covers.
Regards

**Comment by Grunthos** [comment]
Did you read the messages it displayed on upgrade (did they display?). It should have prompted you to locate the old 'bookCatalogue' directory and explained the reasons for the changes. It seems many people just click through these. Do you think that's what happened?

And, yes, the new way of reading the archives is inferior, but that is what Google have forced us to do. We have to use their pathetic excuse for a file browser which we are not in any way able to customize. The only upside (and it's good) is that you can now restore/backup to places like Google Drive.

**Comment by Miecz5** [comment]
Thank you for the quick reply.
Yes, the message appeared (I checked it on the other phone), but ... when I have dozens of applications installed, there are so many updates that the information appears - I admit that I do not pay attention.
I moved the cover photos to a new location and the app read them all correctly. It is also correct when we import data from the archive after the update. Everything is done correctly! :-)

P.S. Since I have already created an account here, I will think about additional suggestions to improve the application, because I use it a little differently than the specified fields, but I got used to it and I have to think about whether and how to specify the need.
PS2. Sorry for the grammar, I'm using a translator...
